samples/openapi3/client/petstore/python-experimental/petstore_api/signing.py

```diff
@@ -126,6 +138,24 @@ def __init__(self, key_id, signing_scheme, private_key_path,
         self.private_key_path = private_key_path
         self.private_key_passphrase = private_key_passphrase
         self.signing_algorithm = signing_algorithm
+        self.hash_algorithm = hash_algorithm
```

Comment:
Where is self.hash_algorithm used other than these exception raising checks?
If it is only used for exception raising, why assign it as the self.hash_algorithm instance variable?

Reply:
It's used while creating the digest.

Reply:
Ah, thanks I missed that